### PR TITLE
#VP-214 templates as resources

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/Filters/TranslationFilter.cs [hunk 1]
```diff
@@ -11,10 +11,17 @@ public static string Translate(TemplateContext context, string key, string langu
             var result = key;
 
             var localizationService = (ITranslationService)context.GetValue(new ScriptVariableGlobal(nameof(NotificationScriptObject.TranslationService)));
+
             if (string.IsNullOrEmpty(language))
             {
                 language = context.GetValue(new ScriptVariableGlobal(nameof(NotificationScriptObject.Language)))?.ToString();
             }
+
+            if (!string.IsNullOrEmpty(language))
```
Comment (tatarincev) [inline Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/Filters/TranslationFilter.cs]: Why do we need this a key transformation here? We don't use language as par of key in our translation tables

File: Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/LiquidTemplateRenderer.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public async Task<string> RenderAsync(string stringTemplate, object model, strin
             };
             var scriptObject = AbstractTypeFactory<NotificationScriptObject>.TryCreateInstance();
             scriptObject.Import(model);
-            scriptObject.Language = language;
+            scriptObject.Add(nameof(NotificationScriptObject.Language), language);
```
Comment (tatarincev) [inline Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/LiquidTemplateRenderer.cs]: This line also seems  redundant, `scriptObject` has a logic that allows to access it type properties without this constructions.

File: Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/Filters/TranslationFilter.cs (hunk 1 above)
Comment (kostyrin) [reply Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/Filters/TranslationFilter.cs]: no need. you right!

File: Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/LiquidTemplateRenderer.cs (hunk 2 above)
Comment (kostyrin) [reply Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/LiquidTemplateRenderer.cs]: https://github.com/lunet-io/scriban/blob/5718821436a31c0e212452921fbee3fb045aebca/src/Scriban/Runtime/ScriptObject.cs#L21 no, it's need to

**Pull request merged**